node/as/json.js

```diff
@@ -39,7 +38,8 @@ function TChannelJSON(options) {
 
     var self = this;
 
-    self.logger = options && options.logger || NullLogtron();
+    // lazily populated from tchannel
+    self.logger = null;
```

Comment:
can you please require `../null-logger` and default to it? then drop the `if (self.logger)` guarding.